bin/taiko.js

```diff
@@ -28,6 +28,12 @@ async function exitOnUnhandledFailures(e){
     }
 }
 
+async function loadScript() {
+    await runFile(process.argv[3]);
```

Comment:
this might break when user tries to use `observe` command in a different order, like `taiko -o --load script.js`

Reply:
@NivedhaSenthil what does  `observe ` command do? i tried from master and it always looks for .js file

Reply:
Current use of `observe` command is `taiko script.js --observe`,  we send --load before the script.js.

If a user wants to use `observe` with load scrip then `taiko --load script.js --observe` this works as expected

Reply:
restricting user to follow order for command line args is not very user friendly, we should be considering some CLI framework since the command line args are growing in runner 
like http://yargs.js.org/